src/components/organisms/WithNavigationBar/WithNavigationBar.scss

```diff
@@ -4,8 +4,69 @@
   padding-top: $navbar-height;
   padding-bottom: $footer-height;
 }
+
 .navbar-margin.fullscreen {
   height: 100%;
   width: 100%;
   overflow: auto;
 }
+
+.nav-logos {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav-sparkle-logo,
+.nav-sparkle-logo_small {
+  position: relative;
+  margin-right: 12px;
+  &:hover {
+    width: 120px;
+  }
+  &.nav-sparkle-logo_small {
+    display: none;
+  }
+  div {
```

Comment:
We shouldn't be directly styling html elements (eg. `div`) like this. Is there any reason we need this 'inner child' at all? Can we just move these styles to the 'root' of `.nav-sparkle-logo`?

Reply:
Moved. And renamed.